keith-vscode/package.json

```diff
@@ -561,8 +561,9 @@
     "lint": "eslint .",
     "build": "webpack --mode production",
     "watch": "webpack --watch",
-    "package": "vsce package --yarn -o keith-vscode.vsix",
-    "vscode:prepublish": "yarn run build"
+    "package": "vsce package --yarn -o keith-vscode.vsix && yarn run copy-webview",
```

Comment:
If you define a valid `package` script in the webview package.json, calling that script on the top level (as should be done for building the system) will automatically include building the webview code.